jobs/JREGIONAL_RUN_POST

```diff
@@ -138,16 +138,14 @@ Call to ex-script corresponding to J-job \"${scrfunc_fn}\" failed."
 #
 #-----------------------------------------------------------------------
 #
-if [ "${FCST_LEN_HRS}" = "-1" ]; then
-  for i_cdate in "${!ALL_CDATES[@]}"; do
-    if [ "${ALL_CDATES[$i_cdate]}" = "${PDY}${cyc}" ]; then
-      FCST_LEN_HRS="${FCST_LEN_CYCL_ALL[$i_cdate]}"
-      break
-    fi
-  done
+if [ "${NUM_FCST_LEN_CYCL}" -gt "1" ]; then
```

Comment:
This new variable does not need to be added because the same can be accomplished by bash using the existing variables, which reduces unnecessary complexity. For example:

```
if [ ${#FCST_LEN_CYCL[@]} -ne 1 ]; then
```

Reply:
@christinaholtNOAA, you're correct. I'll modify it soon.